tests/unit/lms/views/exceptions_test.py

```diff
@@ -6,102 +6,108 @@
 from lms.models import ReusedConsumerKey
 from lms.resources import LTILaunchResource
 from lms.resources._js_config import JSConfig
-from lms.services import HAPIError
+from lms.services import HAPIError, SerializableError
 from lms.validation import ValidationError
 from lms.views.exceptions import ExceptionViews
 
 
 class TestExceptionViews:
-    def test_notfound(self, assert_response, pyramid_request):
+    def test_notfound(self, pyramid_request):
         exception = httpexceptions.HTTPNotFound()
 
         template_data = ExceptionViews(exception, pyramid_request).notfound()
 
-        assert_response(template_data, 404, message="Page not found")
+        assert pyramid_request.response.status_int == 404
+        assert template_data == {"message": "Page not found"}
```

Comment:
I've changed this because:

* Using the helper was more code than not using it
* It made reading tests harder
* It was causing all sorts of coverage issues with the conditionality in it